waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/mapping/service/impl/YamlFederatedMetaStoreStorageTest.java

```diff
@@ -161,6 +162,36 @@ public void loadFederationMultipleFederations() throws Exception {
     assertThat(storage.getAll().get(2), is(metaStore));
   }
 
+  @Test
+  public void loadFederationMappedDatabasesAndTables() throws Exception {
+    File f = dataFolder.getFile("mapped-tables.yml");
```

Comment:
f=file

Reply:
Ok, I'll change it in the previous tests too